wolfssl/internal.h

```diff
@@ -1342,19 +1342,26 @@ enum Misc {
     DTLS_POOL_SZ             = 255,/* allowed number of list items in TX pool */
     DTLS_EXPORT_PRO          = 165,/* wolfSSL protocol for serialized session */
     DTLS_EXPORT_STATE_PRO    = 166,/* wolfSSL protocol for serialized state */
-    DTLS_EXPORT_VERSION      = 4,  /* wolfSSL version for serialized session */
```

Comment:
Should we keep some of the old DTLS names for anyone who might be using these?

Reply:
Users should not be including internal.h. I feel that it should break for them if they are, and we can provide free help to them to include the right header files and help out.